src/pages/index.tsx

```diff
@@ -211,6 +213,11 @@ const IndexPage = () => {
                   Alerts & Notifications
                 </a>
               </li>
+              <li>
+                <a href="/product/session-replay/">
+                  Session Replay
+                </a>
+              </li>
```

Comment:
New footer link on the homepage: http://localhost:3000/

<img width="1110" alt="SCR-20221205-no0" src="https://user-images.githubusercontent.com/187460/205780233-6f735cf9-745b-4cd8-b361-749fabf26c70.png">

Reply:
I wonder if we can move that to a more 'prime location'.
How about we swap it with `Sentry-CLI` 😏 
Or another arrangement that makes more sense?

Reply:
totally agree. this was an 'empty' spot in the grid, but the columns/order is hard-coded as html, so we can move things around.

Might need to get buy-in from some other team. Lets split this off into a separate change.

Reply:
I'll rip this change out of this PR, replaced with : https://github.com/getsentry/sentry-docs/pull/5911